transformer_engine/pytorch/csrc/comm_gemm_overlap.h

```diff
@@ -901,6 +907,10 @@ struct UbufP2PCommOverlap : torch::CustomClassHolder, UbufBase {
         // GEMM
         torch::Tensor output_chunk = torch::from_blob(
             output_ptr + (send_chunk_id * output_chunk_bytes), {n_chunk, m}, D.options());
+        if (do_gelu) {
+          pre_gelu_out = torch::from_blob(
+              pre_gelu_out_ptr + (send_chunk_id * aux_chunk_bytes), {n_chunk, m}, pre_gelu_out.options());
```

Comment:
Fixing linter complaint:
```suggestion
              pre_gelu_out_ptr + (send_chunk_id * aux_chunk_bytes),
              {n_chunk, m},
              pre_gelu_out.options());
```